core/power/impl/power_table_error.cpp

```diff
@@ -13,9 +13,5 @@ OUTCOME_CPP_DEFINE_CATEGORY(fc::power, PowerTableError, e) {
       return "PowerTableError: miner not found";
     case PowerTableError::NEGATIVE_POWER:
       return "PowerTableError: power cannot be negative";
-    case PowerTableError::UNKNOWN:
-      break;
   }
```

Comment:
This return is for the case when the numerical value of an enum object is neither of its constants.
```suggestion
  }
  return "Unknown error";
```